extensions/vscode/webviews/homeView/vite.config.ts

```diff
@@ -18,13 +18,6 @@ export default defineConfig({
       src: fileURLToPath(new URL("./src", import.meta.url)),
     },
   },
-  css: {
-    preprocessorOptions: {
-      scss: {
-        api: "modern-compiler",
-      },
-    },
-  },
```

Comment:
This was removed as this is now the default behavior. https://vite.dev/guide/migration#sass-now-uses-modern-api-by-default